pom.xml

```diff
@@ -281,12 +281,12 @@
         <dependency>
             <groupId>org.opensearch</groupId>
             <artifactId>opensearch</artifactId>
-            <version>2.8.0</version>
+            <version>2.11.1</version>
```

Comment:
Kan vi oppgradere? Er vi nødt til å ha samme versjon som server-instansen [ref. alarmen om versjonsavvik](https://grafana.nav.cloud.nais.io/d/zOfzwJVZk/tekniske-metrikker?orgId=1&refresh=1m&viewPanel=274)? Eller gjelder det bare major-versjon? Mulig @dragutin-nav har oversikt!

Reply:
Å, takk for kommentar! Jeg skal se om jeg finner ut av det, hvis ikke får jeg spørre Drago på mandag.

Reply:
For meg ser det ut som det som sammenlignes i koden er disse to versjonene (og der har jeg endret begge):
<dependency>
            <groupId>org.opensearch</groupId>
            <artifactId>opensearch</artifactId>
            <version>2.11.1</version>
        </dependency>
        <dependency>
            <groupId>org.opensearch.client</groupId>
            <artifactId>opensearch-rest-high-level-client</artifactId>
            <version>2.11.1</version>
        </dependency>

Reply:
Og det er major-versjoner som vil slå ut. Nå var det uansett nødvendig å bumpe begge samtidig, så de er på helt samme versjon.